acorn/src/statement.js

```diff
@@ -489,17 +496,17 @@ pp.parseForIn = function(node, init) {
 
 // Parse a list of variable declarations.
 
-pp.parseVar = function(node, isFor, kind) {
+pp.parseVar = function(node, isFor, kind, allowMissingInitializer) {
```

Comment:
Changing the behavior of this method when called without the new argument is, again, going to break existing plugins.

Reply:
Could you explain why it will break existing plugins? The follow wont cause error.
```
// in acorn
class Person {
  constructor() {
    this.name = 'sss'
  }
}

const pp = Person.prototype

pp.speak = function(word1, word2) {
  console.log(word1)
}

// in plugins
Person.prototype.speak('123')
```

Reply:
Indeed, on closer look it seems like the behavior when the parameter isn't passed doesn't change.

Reply:
Yes, the parameter will be empty while u dont pass it. That wont change the original behavior.